medical_physician/wizard/medical_physician_unavailable.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# © 2015 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalPhysicianUnavailableWizard(models.TransientModel):
+    _name = 'medical.physician.unavailable.wizard'
+    _description = 'Medical physicians unavailable wizard'
+    physician_id = fields.Many2one(
+        'medical.physician', 'Physician', required=True
+        )
+    date_start = fields.Datetime(
+        string='Start', default=fields.date.today, required=True
+        )
+    date_end = fields.Datetime(
+        string='End', default=fields.date.today, required=True
+        )
+    institution_id = fields.Many2one(
+        'res.partner', 'Medical Center', select=True,
+        domain=[('is_institution', '=', True)]
+        )
+
+    def action_cancel(self, cr, uid, ids, context=None):
```

Comment:
This can be removed. It's not being used, and has already been replaced in view for `button special="cancel"`